src/js/components/state/topFive/TopFiveRow.jsx

```diff
@@ -15,20 +17,24 @@ const TopFiveRow = (props) => {
     const percentValue = (props.data._amount / props.total) * 100;
 
     const percent = isNaN(percentValue) ? '--' : `${Math.round(percentValue * 100) / 100}%`;
+    const formattedAmount = MoneyFormatter.formatStateValues(props.data._amount);
```

Comment:
See later comment about formatting logic. This should be done in the model so that `{props.data.amount}` is already formatted.